JobRecruiting/JobRecruiting/src/main/java/com/recruit/jobrecruiting/util/Util.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.recruit.jobrecruiting.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author DENISA
+ */
+public class Util {
+
+    public static <T> List<T> detachEntities(List<Detachable> inputArray) {
```

Comment:
la ce se foloseste?

Reply:
pentru a transforma din entity class in "details" (pojo)  - am facut ceva mai general ca sa nu cipem for-ul ala de fiecare data , asa trebe numa sa te ocupi de creatia clasei "details" (am pus exemplu in jobpost)